CMakeLists.txt

```diff
@@ -17,6 +17,7 @@ raf_option(RAF_USE_NCCL "Build RAF with NCCL. Option: [ON/OFF]" OFF)
 raf_option(RAF_USE_CUBLAS "Build RAF with cuBLAS. Option: [ON/OFF]" OFF)
 raf_option(RAF_USE_GTEST "Build cpptests for RAF. Option: [ON/OFF]" OFF)
 raf_option(RAF_USE_SANITIZER "Build RAF with sanitizer. Option: [OFF/ASAN/MSAN/TSAN/UBSAN]" OFF)
+raf_option(RAF_INSTALL_DEV "Install RAF dev header files. Option: [ON/OFF]" OFF)
```

Comment:
Is there any drawback if we always include the header files? Otherwise we could remove this config for simplification

Reply:
Yeah. What I meant was could we just remove this configure and make it always ON?

Reply:
That's actually good point, let me check.